### ISSUE TITLE
LaTeX table doesn't respond to updated configure

### ISSUE THREAD

**Issue by caitlinadams** [Samreay/ChainConsumer#65]
Hi Sam,

I'm having an issue with the `analysis.get_latex_table` function. Given that I occasionally have skewed parameter distributions, I've taken to quoting both the maximum likelihood value and the cumulative value (with errors). Ideally, I'd like to output tables for both so that I can easily transfer them to my thesis without needing to re-run the analysis after manually changing the statistic. However, it looks like `analysis.get_latex_table` only pays attention to the statistic that is configured first. 

Here's the minimum working example:
```
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import skewnorm
from chainconsumer import ChainConsumer

np.random.seed(0)
data = skewnorm.rvs(5, size=(1000000, 2))
parameters = ["$x$", "$y$"]

c = ChainConsumer()
c.add_chain(data, parameters=parameters)

c.configure(statistics="max")
table_max = c.analysis.get_latex_table(caption="Results for the tested models", label="tab:max")
print(table_max)

c.configure(statistics="cumulative")
table_cumulative = c.analysis.get_latex_table(caption="Results for the tested models", label="tab:cumulative")
print(table_cumulative)
```

If you run this, both tables show the results from the `max` statistic. If you swap the order (run the cumulative table first), both tables show the results from the `cumulative` statistic instead.

Is there a way to get the function to update the statistic before making the table? I had a look through the code but couldn't quite figure out how to do it!

Sorry for all the issues recently -- you can tell I'm up to the results section of my thesis!
Caitlin

**Comment by Samreay** [comment]
Hmm yeah, this is not intended behaviour. Let me have a dig around this arvy and see whats causing it :)

**Comment by Samreay** [comment]
Should now be fixed in v0.26.2
